buildSrc/getNativeCacheLocation.js

```diff
@@ -0,0 +1,18 @@
+import fs from "node:fs/promises"
```

Comment:
we should document this a bit. "this script outputs module version as found in package-lock.json. This is useful in CI to know the version of the package we expect before we install the dependencies".